src/k8s-configuration/setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '1.0.0'
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'License :: OSI Approved :: MIT License',
+]
+
+# TODO: Add any additional SDK dependencies here
+DEPENDENCIES = ["pycryptodome~=3.9.8", "paramiko~=2.6.0", "cryptography>=2.3.1,<3.0.0"]
```

Comment:
[paramiko](https://github.com/Azure/azure-cli/blob/1f875891e6cb858a1d02da8fede885ad7e348832/src/azure-cli-core/setup.py#L57) is a dependency of `azure-cli-core`, it should be removed here, then you don't need to pin `cryptography` which is a dependency of `paramiko`.

Reply:
Removed dependencies